SimplenoteIntents/IntentHandler.swift

```diff
@@ -2,18 +2,62 @@ import Intents
 import CoreData
 
 class IntentHandler: INExtension {
+    let coreDataManager: CoreDataManager
+
+    override init() {
+        do {
+            self.coreDataManager = try CoreDataManager(StorageSettings().sharedStorageURL, for: .intents)
+        } catch {
+            fatalError()
+        }
+        super.init()
+    }
+
+
     override func handler(for intent: INIntent) -> Any {
+        NSLog("failed to return handler")
```

Comment:
We should prob drop this one (or format this a bit more, like `[IntentHandler] Failed to return handler` ?

P.s.: Does this imply the handler failed? since it's returning `self`

Reply:
I think this got left in there cause I was doing some testing on the previous glitches and wasn't sure where the issue was yet.  Will drop it